src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/geo/GEOMiner.java

```diff
@@ -42,22 +48,33 @@
  *
  * @see GEOResource
  */
-public class GEOMiner extends AContainer implements RecipeDisplayItem, HologramOwner {
+public class GEOMiner extends SlimefunItem implements RecipeDisplayItem, EnergyNetComponent, InventoryBlock, HologramOwner, MachineProcessHolder<MiningOperation> {
 
     private static final int[] BORDER = { 0, 1, 2, 3, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 26, 27, 35, 36, 44, 45, 53 };
     private static final int[] OUTPUT_BORDER = { 19, 20, 21, 22, 23, 24, 25, 28, 34, 37, 43, 46, 47, 48, 49, 50, 51, 52 };
     private static final int[] OUTPUT_SLOTS = { 29, 30, 31, 32, 33, 38, 39, 40, 41, 42 };
+
     private static final int PROCESSING_TIME = 14;
+    private static final int ENERGY_CONSUMPTION = 24;
```

Comment:
Having this as a static field breaks anything that wants to extend GeoMiner and change consumption, it should be provided in constructor or smth